core/code/search.js

```diff
@@ -401,3 +401,50 @@ addHook('search', function(query) {
   $.getJSON(NOMINATIM + encodeURIComponent(query.term) + viewbox + bounded, onQueryResult.bind(null, true));
 });
 
+// search for guid
+window.search.addResult = function (query, data) {
```

Comment:
This function is very similar to code starting at line 270...
We should extract the part from line 270 to a function and call it instead to get the same visuals for both "portal results"

Reply:
No longer. Function needed GUID as third parameter, as guid is not part of data.

Reply:
The code is still duplicate and we could have one method instead of 2 slightly different (one is addResilult and second is the part from line 270 - that's not method but it's very much the same as the fubction we added... With some special formatting...

If we don't mind different formatting for the same results, then it's fine 😉